src/frontend/screens/Settings/components/WinePrefixesBasePath.tsx

```diff
@@ -19,20 +21,59 @@ const WinePrefixesBasePath = () => {
     'defaultWinePrefix',
     ''
   )
+  const [lastValidPrefix, setLastValidPrefix] = useState(defaultWinePrefix)
+
+  function handlePathChange(val: string) {
+    setDefaultWinePrefix(val)
+    if (val) {
```

Comment:
I'm not sure whether a simple truthiness check makes sense here. Sure, if the user highlights the entire path and then deletes it, this'll work. More likely however (especially considering virtual keyboards, e.g. on the Deck), they're gonna backspace one-by-one, causing a restore operation to just re-insert the `/`. This of course won't work as a prefix path either

The easiest way to resolve this would be to not store the "last valid" value at all, instead just restoring it to the same path a new install would use:

https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/blob/47d3075ca1c53a8e009b6a504a839a3143e11d0e/src/frontend/screens/Library/components/InstallModal/WineSelector/index.tsx#L70-L73

Reply:
Sorry if this is a silly question but I don't see where to find the original `defaultWinePrefix`? In the code you linked the `suggestedWinePrefix` is built partly the `title` or `appName` from `WineSelector`'s props, which we don't have in the component we're working on... Is there something like a constants file that the initial value comes from?